sentry-compose-helper/src/jvmMain/java/io/sentry/compose/viewhierarchy/ComposeViewHierarchyExporter.java

```diff
@@ -99,6 +105,21 @@ private static void setTag(
             }
           }
         }
+      } else if ("androidx.compose.ui.platform.TestTagElement".equals(type)
```

Comment:
is it possible to test it with robolectric/instrumented tests to avoid regressions in future?